src/pudl/package_data/eia923/column_maps/financial_information.csv

```diff
@@ -0,0 +1,31 @@
+year_index,2008,2009,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020,2021,2022
+report_year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year
+plant_id_eia,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id
+fly_ash_collection_operation_maintenance_expense_thousand_dollars,fly_ash_collection_o_m_expense,fly_ash_collection_o_m_expense,fly_ash_collection_o_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense_thousand_dollars,fly_ash_collection_o_and_m_expense_thousand_dollars,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense,fly_ash_collection_o_and_m_expense
```

Comment:
Do we care/have a meaningful distinction between `cost` and `expense` as column suffixes? Maybe @cmgosnell or @zaneselvans

Reply:
here is a nice little article that details the [cost vs expense difference](https://www.thebalancemoney.com/cost-vs-expense-what-is-the-difference-3974582).

if i understand correctly most maintenance = expense.